packages/internet_detector.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,37 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'internet_detector'
+$category = 'Utilities'
+
+# Install dependency for windows api
+VM-Pip-Install "pywin32"
+
+# Create tool directory
+$toolDir = Join-Path ${Env:RAW_TOOLS_DIR} $toolName
+New-Item -Path $toolDir -ItemType Directory -Force -ea 0
+VM-Assert-Path $toolDir
+
+$packageToolDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+# Move images to %VM_COMMON_DIR% directory
+$imagesPath = Join-Path $packageToolDir "Images"
+Move-Item "$imagesPath\*" ${Env:VM_COMMON_DIR} -Force
+
+# Get path to pythonw for executing Python without terminal popup
+$executablePath = (Get-Command pythonw).Source
+$filePath = Join-Path $packageToolDir "$toolName.pyw"
+
+VM-Install-Shortcut $toolName $category $executablePath -arguments $filePath
+
+# Create scheduled task for tool to run every 2 minutes.
+$action = New-ScheduledTaskAction -Execute $executablePath -Argument "$filePath"
+$trigger = New-ScheduledTaskTrigger -Once -At (Get-Date) -RepetitionInterval (New-TimeSpan -Minutes 2)
+Register-ScheduledTask -Action $action -Trigger $trigger -TaskName 'Internet Detector' -Force
+
+# Move updated `default.ini` with tool added to ProcessBlackList to FakeNet Config Directory
+$fakenetConfigDir = Join-Path ${Env:UserProfile} "Desktop\fakenet_logs.lnk"
+if (Test-Path $fakenetConfigDir) { # Check if the shortcut exists
+    $targetPath = (New-Object -ComObject WScript.Shell).CreateShortcut($fakenetConfigDir).TargetPath
+    Move-Item "$packageToolDir\default.ini" -Destination $targetPath
```

Comment:
I think you are doing something wrong moving the file, as the file I see in my fakenet default configuration after installing this package does not have your changes. Please test this out @emtuls!

Also, I would copy instead of moving, so that we have a copy of the file in case it is needed it.